### ISSUE TITLE
Paper?

### ISSUE THREAD

**Issue by kirefu** [aakhundov/deep-german#1]
Hi @aakhundov 

Do you have a paper describing your work here? 

Best,

**Comment by aakhundov** [comment]
No, we never wrote a paper about the results. A succinct description of the work is provided in the README.md. Otherwise, please look into the code for details.
